cves/kernel/CVE-2022-27223.yml

```diff
@@ -114,7 +109,7 @@ upvotes_instructions: |
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes:
+upvotes: 
```

Comment:
I would give my interest in this vulnerability a two out of ten since it seems to be just a forgotten check on an array.